src/RuleSet/Sets/PhpCsFixerSet.php

```diff
@@ -109,6 +109,7 @@ public function getRules(): array
             'phpdoc_add_missing_param_annotation' => true,
             'phpdoc_no_empty_return' => true,
             'phpdoc_order_by_value' => true,
+            'phpdoc_to_comment' => false,
```

Comment:
case 4

ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6680#discussion_r1016891055

I would actually suggest to NOT change the ruleset, but rather play with `.php-cs-fixer` config to temporarily exclude that rule/file, and then fix the rule to handle that case [separated PR]

Reply:
Fair enough, will do.